### PR TITLE
add full 2023 roster, add a bunch of pictures

### PR DESCRIPTION
**Pull request by a14glick**
Finally Updated! Still need some pictures.

### CODE REVIEW

File: content/roster/2023/_index.md [hunk 1]
```diff
@@ -3,9 +3,19 @@ title: "2023 Roster"
...
-showroster: false
-showpractice: false
+showpreseason: false
+showcaptains: true
+showcoach: true
+showroster: true
+showpractice: true
 ---
 
+#### Captain:
```
Comment (brendanwallace) [inline content/roster/2023/_index.md:13]: these changes don't do anything (this stuff is the "content" of the _index page but I'm not actually using it in the rendering)

please delete from #### Captain: onwards

File: layouts/roster/roster.html [hunk 2]
```diff
@@ -35,7 +35,21 @@ <h3>Captains</h3>
         </div>
         {{ end }}
 
-        
+
```
Comment (brendanwallace) [inline layouts/roster/roster.html:38]: nice :)

**Review by brendanwallace — CHANGES_REQUESTED** [review]
It all looks good except for the unnecessary changes to _index.md!

File: layouts/roster/roster.html (hunk 2 above)
Comment (a14glick) [reply layouts/roster/roster.html:38]: thank!

**Review by brendanwallace — APPROVED** [review]
LGTM

**Pull request merged**